src/traces/contour/constraint_value_defaults.js

```diff
@@ -9,14 +9,13 @@
 
 'use strict';
 
-var constraintMapping = require('./constraint_mapping');
 var isNumeric = require('fast-isnumeric');
+var COMPARISON_OPS2 = require('../../constants/filter_ops').COMPARISON_OPS2;
```

Comment:
Looks like `constraint_value_defaults.js` is only required in `constraint_defaults.js`. Would you mind merging them in a single file?

Reply:
ah yes, no need for separate files - https://github.com/plotly/plotly.js/pull/2270/commits/7e837efd67cdd56c5934fbf92d4f8fbbd545c305